localstack/services/stepfunctions/asl/component/state/state_choice/comparison/operator/implementations/numeric.py

```diff
@@ -0,0 +1,139 @@
+from typing import Any
```

Comment:
I really like the more condensed version here instead of splitting it into too many files :+1: